app/views/RoomView/index.js

```diff
@@ -412,9 +412,9 @@ class RoomView extends React.Component {
 		this.setState({ selectedMessage: message, showErrorActions: true });
 	}
 
-	onActionsHide = () => {
+	onActionsHide = (hideAction = false) => {
 		const { editing, replying, reacting } = this.state;
-		if (editing || replying || reacting) {
+		if ((editing || replying || reacting) && !hideAction) {
```

Comment:
You is never consider `editing || replying || reacting`.
It isn't the correct solution, you need to make the changes requested #1610, and then you can remove this `if` case.

Reply:
Sir, I have made the changes in #1610, and I wanted to know that should I close this Pr and push commit to fix this issue in #1610 itself, since the solution relies on the code there?
https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/1610#issuecomment-580195161